snowshu/core/utils.py

```diff
@@ -122,3 +123,9 @@ def remove_dangling_replica_containers() -> None:
     for container in client.containers.list(all=True):
         if 'snowshu_target_' in container.name:
             container.remove(force=True)
+
+
+def generate_unique_uuid_name(name: str) -> str:
+    """Generates a unique name based on name and randomly generated uuid."""
+    _uuid = str(uuid.uuid4()).rsplit('-', maxsplit=1)[-1]
```

Comment:
I'm using only the last part of the UUID for two main reasons: a full UUID is too long and looks awkward, and using just a part of it makes it easier to identify your replicas while still keeping the chance of name conflicts very low.